ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -459,6 +459,6 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "draftCMO",
     "UserRole": "caseworker-publiclaw-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "CRUD"
```

Comment:
why do I need D permissions on the event?

Reply:
Removed this - i was getting the 'no field' error on submit, but i've moved the D permission to casefield caseManagementOrder which Amit uses (he deletes in the controller)